### ISSUE TITLE
Insert PHP

### ISSUE THREAD

**Issue by OlivierFacelina** [OlivierFacelina/Stage-2022#1]
``<?php
     
    require('connexion.php');
 
    $role_user_error = $email_user_error = $password_user_error = $nom_user_error = $prenom_user_error = $active_user_error = "";

    if (isset($_POST['submit'])) {
        $role_user        = checkInput($_POST['role']);
        $email_user              = checkInput($_POST['email']);
        $password_user           = checkInput($_POST['password']); 
        $nom_user           = checkInput($_POST['nom']);
        $prenom_user           = checkInput($_POST['prenom']);
        $active_user           = checkInput($_POST['active']);
        $isSuccess          = true;
        
        if(empty($role_user)) {
            $role_user_error = 'Ce champ ne peut pas être vide';
            $isSuccess = false;
        } 
        if(empty($email_user)) {
            $email_user_error = 'Ce champ ne peut pas être vide';
            $isSuccess = false;
        } 
        if(empty($password_user)) {
            $password_user_error = 'Ce champ ne peut pas être vide';
            $isSuccess = false;
        }
        if(empty($nom_user)) {
            $nom_user_error = 'Ce champ ne peut pas être vide';
            $isSuccess = false;
        }
        if(empty($prenom_user)) {
            $prenom_user_error = 'Ce champ ne peut pas être vide';
            $isSuccess = false;
        }
        if(empty($active_user)) {
            $active_user_error = 'Ce champ ne peut pas être vide';
            $isSuccess = false;
        }
        
        if($isSuccess) {
            $statement = $db->prepare("INSERT INTO utilisateur(role_user,email_user,password_user,nom_user,prenom_user,active_user) VALUES(?,?,?,?,?,?);");
            $statement->execute(array($role_user,$email_user,$password_user,$nom_user,$prenom_user,$active_user));
            header("Location: affichage.php");
        }
    }
    
    function checkInput($data) {
      $data = trim($data);
      $data = stripslashes($data);
      $data = htmlspecialchars($data);
      return $data;
    }
?>

<!DOCTYPE html>
<html>
    <head>
        <title>Burger Code</title>
        <meta charset="utf-8"/>
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet">
        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/js/bootstrap.bundle.min.js"></script>
        <link href='http://fonts.googleapis.com/css?family=Holtwood+One+SC' rel='stylesheet' type='text/css'>
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
        <link rel="stylesheet" href="insert.css">
        <script src="https://kit.fontawesome.com/568733c487.js" crossorigin="anonymous"></script>
    </head>
    
    <body>
        <h1 class="text-logo"><span class="bi-shop"></span> Ajouter un utilisateur <span class="bi-shop"></span></h1>
        <div class="container admin">
            <div class="row">
                <br>
                <form class="form" action="insert.php" role="form" method="post" enctype="multipart/form-data">
                    <br>
                    <div>
                        <label class="form-label" for="role">Role:</label>
                        <input type="text" class="form-control" id="role" name="role" placeholder="Role">
                        <span class="help-inline"><?php echo $role_user_error;?></span>
                    </div>
                    <br>
                    <div>
                        <label class="form-label" for="email">Email:</label>
                        <input type="email" class="form-control" id="email" name="email" placeholder="Email">
                        <span class="help-inline"><?php echo $email_user_error;?></span>
                    </div>
                    <br>
                    <div>
                        <label class="form-label" for="password">Password:</label>
                        <input type="password" class="form-control" id="password" name="password" placeholder="Password">
                        <span class="help-inline"><?php echo $password_user_error;?></span>
                    </div>
                    <br>
                    <div>
                        <label class="form-label" for="nom">Nom:</label>
                        <input type="text" class="form-control" id="nom" name="nom" placeholder="Nom">
                        <span class="help-inline"><?php echo $nom_user_error;?></span>
                    </div>
                    <br>
                    <div>
                        <label class="form-label" for="prenom">Prenom:</label>
                        <input type="text" class="form-control" id="prenom" name="prenom" placeholder="Prenom">
                        <span class="help-inline"><?php echo $prenom_user_error;?></span>
                    </div>
                    <br>
                    <div>
                        <label class="form-label" for="active">Active:</label>
                        <input type="text" class="form-control" id="active" name="active" placeholder="Active">
                        <span class="help-inline"><?php echo $active_user_error;?></span>
                    </div>
                    <br>
                    <div>
                        <input type="submit" name="submit" value="Ajouter" class="btn btn-success ajout">
                        <a class="btn btn-primary" href="index.php"><span class="bi-arrow-left"></span> Retour</a>
                   </div>
                </form>
            </div>
        </div>
    </body>
</html>
